net/knot/Makefile

```diff
@@ -40,75 +41,76 @@ define Package/knot/Default
 	USERID:=knot=5353:knot=5353
 endef
 
-define Package/knot-libdnssec
+define Package/knot-libs
 	$(call Package/knot-lib/Default)
-	TITLE+= DNSSEC library
-	DEPENDS=+libpthread +libgnutls +jansson
-endef
-
-define Package/knot-libknot
-	$(call Package/knot-lib/Default)
-	TITLE+= library
-	DEPENDS=+libpthread +knot-libdnssec
+	TITLE+= common DNS and DNSSEC libraries
+	DEPENDS+=+libgnutls +jansson
 endef
 
 define Package/knot-libzscanner
```

Comment:
This is a little confusing, there's a "libs" but this isn't part of it.

I liked it better when the previous library was called "knot-libdnssec"